eva_vcf_merge/merge.py

```diff
@@ -65,6 +65,8 @@ def common_merge(self, merge_type, vcf_groups, chunk_size=500, resume=True):
             safe_alias = get_valid_filename(alias)
             target_filename = os.path.join(self.output_dir, f'{safe_alias}_merged.vcf.gz')
             shutil.move(merged_filenames[alias], target_filename)
+            shutil.move(merged_filenames[alias] + '.csi', target_filename + '.csi')
+            shutil.move(merged_filenames[alias] + '.tbi', target_filename + '.tbi')
```

Comment:
It might be a good idea to add a test (or modify existing tests) to check for the final index files.

Reply:
Done